models/balancer/ethereum/balancer_ethereum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+        alias ='pools_tokens_weights'
+        )
+}}
+
+SELECT *
+FROM
+(
+        SELECT
+                pool_id,
+                token_address,
+                normalized_weight
+        FROM {{ ref('balancer_v1_ethereum_pools_tokens_weights') }}
+        UNION
```

Comment:
please use `union all` rather than `union`
can you make that change in all models?

Reply:
This is now fixed. @mendesfabio recommended changing the `JOIN` condition so that we don't have duplicate values.